fractions/microprofile/microprofile-jwt/pom.xml

```diff
@@ -24,6 +24,7 @@
   <properties>
     <swarm.fraction.stability>experimental</swarm.fraction.stability>
     <swarm.fraction.tags>MicroProfile,JWT,Security,Web</swarm.fraction.tags>
+    <version.jose4j>0.6.0</version.jose4j>
```

Comment:
Should be in `/build-parent/pom.xml` like everything else?

Reply:
I'm not sure. I suppose that `build-parent` is mostly useful for dependencies which might be shared and this is probably not the case. Also there are few places where we don't use build-parent, e.g. netflix: https://github.com/wildfly-swarm/wildfly-swarm/blob/master/fractions/netflix/pom.xml#L26 or even hardcoded versions in fluentd: https://github.com/wildfly-swarm/wildfly-swarm/blob/master/fractions/fluentd/src/main/resources/modules/org/fluentd/main/module.xml#L7.

@kenfinnigan @heiko-braun What do you think?

Reply:
I don't think it's necessary for all versions of things to be in build-parent, but where it makes the most sense for updating versions of things together.

Beyond that I don't have a particular preference. As you point out @mkouba, there are plenty of places where putting things in build-parent are not followed

Reply:
Ayea. I figured I'd rather ask, but I'm fine with this.